iOS/SDWebImage/build.cake

```diff
@@ -3,17 +3,9 @@
 
 var TARGET = Argument ("t", Argument ("target", "Default"));
 
-var IOS_PODS = new List<string> {
-	"platform :ios, '8.0'",
-	"install! 'cocoapods', :integrate_targets => false",
-	"target 'Xamarin' do",
-	"pod 'SDWebImage', '4.4.6'",
-	"pod 'SDWebImage/MapKit', '4.4.6'",
```

Comment:
Are we dropping the MapKit extensions?

Reply:
These lines are obsolete. The podfile lines with MapKit are some lines below.

Reply:
Ah, I see it now. Just had a blind moment.